app/Template/project_header/views.php

```diff
@@ -1,10 +1,19 @@
 <ul class="views">
+    
+    <?= $this->hook->render('template:project-header:view-switcher-before-ProjectOverview', array('project' => $project, 'filters' => $filters)) ?>
```

Comment:
it would probably be better to keep the naming consistent and use something like `template:project-header:view-switcher-before-project-overview` instead of `template:project-header:view-switcher-before-ProjectOverview`.

Reply:
Done.